.github/pull_request_template.md

```diff
@@ -16,7 +16,36 @@
 - [ ] Add a description of your pull request and instructions for the reviewer to verify your work.
 - [ ] If your pull request is for a specific ticket, link to it in the description.
 - [ ] Stay on point and keep it small so the merge request can be easily reviewed.
-- [ ] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
 - [ ] Tests and linting passes.
-- [ ] Consider updating documentation. If you don't, tell us why.
-- [ ] If relevant, list the environments / browsers in which you tested your changes.
+
+#### Unit tests
+
+- [ ] Not required
+- [ ] Added
+
+#### Documentation
+
+- [ ] Updated
+
+#### Browser testing
+
+- [ ] I have tested in the following browsers and environments:
```

Comment:
I wonder whether we should also add a **Not relevant** option here, since there would be instances where a PR introduces changes that cannnot be tested in a browser.

Reply:
I've amended this now, and added some other 'not required' options.